libmamba/include/mamba/core/output.hpp

```diff
@@ -124,6 +124,7 @@ namespace mamba
         Console& operator=(Console&&) = delete;
 
         static Console& instance();
+        static bool has_instance();
```

Comment:
I would prefer `is_available` as it would match better the functionality and would be less "I know it's a singleton..."